### ISSUE TITLE
InnerTransactions are not contained in a search result.

### ISSUE THREAD

**Issue by 44uk** [nemtech/symbol-sdk-typescript-javascript#783]
1. Expected behavior and actual behavior.
InnerTransactions are not contained in a search result, but `getTransaction` contains them.

2. Steps to reproduce the problem.
reproduce code:
https://runkit.com/nizveyl/innertransactions-are-not-contained-in-a-result

3. Specifications like the version of the project, operating system, or hardware.
symbol-sdk v1.0.0

**Comment by rg911** [comment]
yeah, this is intended for rest. I don't think it is a bug on the sdk. At the moment the `full` aggregate transaction needs 2 queries from the search endpoint. 
1. get the aggregate transaction wrapper. 
2. use the aggregateHash to query the transactions/{hash} to get the full payload. 

Alternatively, you can use the `embedded=true` for in the searchCriteria to get the inner transaction only (no wrapper). However, there are limitations to using this flag in combination with other query params. I think the recipientAddress / signerPublicKey works nicely with embedded, but the tx type (aggComplete) will not work together with embedded.

I agree this limitation is not very convenient and we are planning to improve it (the whole catapult-rest) right now

**Comment by 44uk** [comment]
Ok, thanks. I'm looking forward to improving it!
